tox.ini

```diff
@@ -81,6 +81,18 @@ deps = {[testenv:py34]deps}
 basepython = python3.6
 deps = {[testenv:py34]deps}
 
+[testenv:scurl]
+basepython = python3.6
+deps =
+    {[testenv:py34]deps}
+    Cython
+    git+git://github.com/nctl144/scurl.git
+
+[testenv:scurl27]
+basepython = python2.7
+deps =
+    {[testenv:scurl]deps}
```

Comment:
This will pick up ``{[testenv:py34]deps}`` dependencies, as far as I can see, while we want python 2.7 dependencies here I think.

Reply:
You are definitely right! I just fixed it and the errors on both py27 and py36 are the same now :)

Reply:
That's great that there are no extra errors! 👍